.travis.yml

```diff
@@ -133,22 +138,26 @@ jobs:
 #### PHPUnit tests ###########################
     - &phpunit-tests
       php: 5.6
-      sudo: required
+      addon: {}
       install: composer install --prefer-dist --working-dir=src
       before_script: ./utils/prepare-test -afty
       script:
         - make build-lib VERSIONFILE build-cli
         - src/vendor/bin/phpunit -csrc/phpunit.xml --testsuite="Fossology PhpUnit Test Suite"
-      after_success: php src/vendor/bin/coveralls -vv -x clover.xml
-    - <<: *phpunit-tests
+      after_success: php src/vendor/bin/php-coveralls -vv -o coveralls.json -x clover.xml
+    - &php7-phpunit-tests
+      <<: *phpunit-tests
       php: 7.0
-    - <<: *phpunit-tests
+      script:
+        - make build-lib VERSIONFILE build-cli
+        - phpdbg -qrr src/vendor/bin/phpunit -csrc/phpunit.xml --testsuite="Fossology PhpUnit Test Suite" --colors=always | grep -v 'script>\|c.log'
```

Comment:
Can you please explain why you are using ` | grep -v 'script>\|c.log'`?

Reply:
Just now I saw the explanation on your commit message.

But it will be better if you can provide some more explanation here for others to see.

Reply:
That's the original output message, still working on how to suppress it more properly.
https://travis-ci.com/PeterDaveHello/fossology/jobs/192811862